models/topics.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  let queryString = "SELECT * FROM topics";
+  const extraQueries = [];
+
+  return db
+    .query(queryString, extraQueries)
+    .then(({ rows }) => {
+      return rows;
+    })
+    .catch((err) => {
+      console.log(err);
+      return Promise.reject({
```

Comment:
Think about why you are adding code. For example at what point could there be a wrong input/400 error in this model - there is nothing dynamic about the sql query - it is hard coded to fetch all the topics. Also this "error"/sad path as no testing for it. You could have a 500 error handler that would pick up on any errors from bad code, but there is no need for testing for this.